libs/wingsdk/src/std/duration.ts

```diff
@@ -9,7 +9,7 @@ export class Duration {
    * @returns a new `Duration` representing `amount` Minutes.
    */
   public static fromMinutes(amount: number): Duration {
-    return new Duration(amount * 60);
```

Comment:
No tests were affected by this change?

Reply:
I didn't see any tests for std functions (other than the one I added to sleep)
And since the internals return the same underlying duration, we should be good